src/lib.rs

```diff
@@ -28,7 +28,7 @@
 //! use itertools::interleave;
 //!
 //! for elt in interleave(&[1, 2, 3], &[2, 3, 4]) {
-//!     /* loop body */
+//!        /* loop body */
```

Comment:
Somehow the formatting of the whole file is affected by this commit. Do you think you could push a version without these changes? (Would simplify review quite a bit.)